src/js/script-loader.js

```diff
@@ -834,3 +834,6 @@ var LoaderProtoMethods = {
 Object.keys(LoaderProtoMethods).forEach(function(key) {
     Loader.prototype[key] = LoaderProtoMethods[key];
 });
+
+Loader.prototype.define.amd = {};
```

Comment:
Would this make [this line in the template](https://github.com/julien/liferay-amd-loader/blob/d6cf1879f585c7a791ba7d700c57598f76aa2396/src/template/script-loader.template#L20) unnecessary?